added show of flash message in application.html

Tavbulatov opened the pull request.

Tavbulatov committed:
added show of flash message in application.html

XaoGao commented on .rubocop.yml:
```diff
@@ -20,6 +20,9 @@ Style/StringLiterals:
   Enabled: true
   EnforcedStyle: double_quotes
 
+Style/HashLikeCase:
```
@Tavbulatov  В задаче не было шага вносить изменения в rubocop может эта проверка нужна в других местах

XaoGao commented on app/helpers/application_helper.rb:
```diff
@@ -4,4 +4,13 @@ def title(page_title)
   end
+
+  def flash_message_css_class(flash_type)
+    case flash_type
```
@Tavbulatov  Собственно ни кто не мешает написать так как хочет rubocop тут или локально отключить проверку, но лучше делать как просят стили

XaoGao commented on app/views/layouts/_flash.html.slim:
```diff
@@ -0,0 +1,2 @@
+- flash.each do |key, message|
```
тут может быть что flash будет nil? если да то будем получать ошибку, давай лучше в начале сделаем проверку на то что объект есть

XaoGao commented on app/views/layouts/application.html.slim:
```diff
@@ -10,6 +10,7 @@ html
     = javascript_importmap_tags
   body
     .container
+      = render partial: "layouts/flash"
```
@Tavbulatov header  - это навбар тут, лучше ошибки показывать ниже

XaoGao reviewed (changes requested):
@Tavbulatov отставил замечания

Tavbulatov committed:
fixing some bugs

	изменено:      .rubocop.yml
	изменено:      app/helpers/application_helper.rb
	изменено:      app/views/layouts/_flash.html.slim
	изменено:      app/views/layouts/application.html.slim

XaoGao reviewed (commented):
@Tavbulatov approve
